app/templates/detail_tender.html

```diff
@@ -137,6 +129,60 @@
 
 </div>
 
+<div class="table_detail" >
+
+  {% if tender.awards.all %}
```

Comment:
You can use the {% empty %} here instead of this "if":

https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#for-empty